src/platforms/node/common/configuration/async-context.mdx

```diff
@@ -0,0 +1,40 @@
+---
+title: Async Context
+sidebar_order: 80
+description: "Learn more about how to isolate Sentry scope and breadcrumbs across requests."
+---
+
+You can use the `runWithAsyncContext` method to isolate Sentry scope and breadcrumbs to a single request if you are using SDK v7.48.0 or higher. This is useful if you are finding that breadcrumbs and scope are leaking across requests.
+
+```js
+const Sentry = require("@sentry/node");
+
+function requestHandlerMiddleware(req, res, next) {
+  // Any breadcrumbs or tags added will be isolated to the request
+  return Sentry.runWithAsyncContext(
+    () => {
+      return next(req, res);
+    },
+    { emitters: [req, res] }
```

Comment:
Should we explain what users should pass as `emitters` and what they are used for?

Reply:
I can add a note.

Reply:
Actually I decided to remove this, since 99.9% of users won't need it. We can add additional notes if users ask for it.